README.md

```diff
@@ -18,6 +18,20 @@ $ yarn
 $ yarn start
 ```
 
+3. The local web server will be accessible from (http://localhost:8080)[http://localhost:8080].
+
+#### Docker
+
+If you'd like to use Docker, there's a Docker Compose setup that you can use:
```

Comment:
Add a link to docker compose docs?

Reply:
Added a link to both docker and docker compose overview pages